testsuites/CBLTester/System_test_multiple_device/test_peer_to_peer.py

```diff
@@ -50,7 +50,8 @@ def test_peer_to_peer_1to1_valid_values(params_from_base_test_setup, server_setu
         db_obj_client.create_bulk_docs(num_of_docs, "replication", db=cbl_db_client, channels=channels)
 
     # Now set up client
-    repl = peerToPeer_client.client_start(host=server_host, server_db_name=db_name_server, client_database=cbl_db_client, continuous=continuous, replication_type=replicator_type, endPointType=endPointType)
+    repl = peerToPeer_client.configure(host=server_host, server_db_name=db_name_server, client_database=cbl_db_client, continuous=continuous, replication_type=replicator_type, endPointType=endPointType)
```

Comment:
I hope couple of tests under this file are tested as there are some changes happened on p2p start and stop server

Reply:
just wondering, is there any need to split the flow into two methods? Instead of using 2 statements, why can we have it in one method? Is there any requirement to have in 2 methods?

Reply:
we need to add filters and listener to config. We add them only in rare cases. I've ran all the tests for P2P after modifying API. They all are passing